### PR TITLE
Feature: Added upload complete icon.

### LINKED ISSUE #204: Add confirmation check mark to the image upload animation
There should be :white_check_mark: when the upload finish sucessfully.

This is how it looks now in CKEditor 5:

before

This is how the animation looks like in Letters (with check mark):

after

Without it, upload looks incomplete.

### LINKED ISSUE #5147: Add confirmation check mark to the image upload animation
There should be :white_check_mark: when the upload finish sucessfully.

This is how it looks now in CKEditor 5:

before

This is how the animation looks like in Letters (with check mark):

after

Without it, upload looks incomplete.

### PR DESCRIPTION
**Pull request by oskarwrobel**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Added upload complete icon. Closes ckeditor/ckeditor5#5147.

### CODE REVIEW

File: src/imageupload/imageuploadprogress.js [hunk 1]
```diff
@@ -193,6 +201,21 @@ function _hideProgressBar( viewFigure, writer ) {
...
+//
+// @param {module:engine/view/containerelement~ContainerElement} viewFigure
+// @param {module:engine/view/writer~Writer} writer
+// @param {module:engine/view/view~View} view
+function _showCompleteIcon( viewFigure, writer, view ) {
+	const completeIcon = new UIElement( 'div', { class: 'ck-image-upload-finish' } );
+
+	writer.insert( ViewPosition.createAt( viewFigure, 'end' ), completeIcon );
+
+	setTimeout( () => {
```
Comment (oskarwrobel) [inline src/imageupload/imageuploadprogress.js]: https://github.com/ckeditor/ckeditor5-image/issues/206

**Comment by Mgsy** [comment]
The animation doesn't display properly in Edge.

bug_cke5

**Comment by dkonopka** [comment]
@mgsy ☝️ It's related to `calc()` in the animation https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12872907/, anyways even with "hard-coded" values, there are some issues with `transform-origin`.

I'm not so happy about this, but probably we should drop this animation in check icon..

File: theme/imageuploadicon.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+:root {
+	--ck-image-upload-icon-size: 1.5em;
+	--ck-image-upload-icon-width: 2px;
+
+	--ck-color-image-upload-icon: hsl(0, 0%, 100%);
+	--ck-color-image-upload-icon-background: hsl(120, 100%, 27%);
+}
+
+.ck-image-upload-complete-icon {
```
Comment (oleq) [inline theme/imageuploadicon.css]: I see the wireframe styles (display, position) mixed up with pure visual styles (background, animation) here. The later should be moved to Lark.

**Review by Mgsy — APPROVED** [review]
Everything works fine.

**Comment by pjasiun** [comment]
I also see that green check is bigger and is shown longer than before. It looks much heavier than before. I remember that we had the discussion that is should not be there at all, that the users should have a feeling that the image is loaded in the first place, so the subtle confirmation is a compromise which takes some time. Can we only bring what we had in Letters in this ticket (the same size and time, maybe different color), and move the discussion about changing it to a separate ticket, if needed?

**Pull request merged**